lib/active_force/bulk/records.rb

```diff
@@ -16,10 +16,25 @@ def to_csv
       end
 
       def self.parse_from_attributes(records)
+        # Sorting ensures that the headers line up with the values for the CSV
         headers = records.first.keys.sort.map(&:to_s)
-        data = records.map { |r| r.sort.pluck(-1) }
+        data = records.map do |r|
+           r.transform_values { |v| transform_value_for_sf(v) }.sort.pluck(-1)
+        end
         new(headers: headers, data: data)
       end
+
+      # SF expects a special value for setting a column to be NULL.
+      def self.transform_value_for_sf(value)
+        case value
```

Comment:
🥇 
Dp you wanna store the '#N/A' value in any constant ?

Reply:
Hadn't thought about it but it sounds like a good suggestion to me, thanks!